client/components/Cart.js

```diff
@@ -1,111 +1,114 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { moneyFormatter } from '../utils';
 import { Link } from 'react-router-dom';
-import ProductInCart from './ProductInCart';
 import {
-  fetchCart,
-  quantityOfProducts,
-  addToCart,
   removeFromCart,
+  addQuantity,
+  subtractQuantity,
 } from '../store/redux/cart';
-import { fetchProducts } from '../store/redux/products';
 
-class Cart extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      cart: []
-    }
-    this.handleChange = this.handleChange.bind(this);
-    this.handleClick = this.handleClick.bind(this);
+class Cart extends Component {
+  _removeFromCart(id) {
```

Comment:
In classes, the `_` prefix to a function is meant to state a thing in a class is `private` - this is common across languages, and not specifically a JS thing.